lib/src/bindings/bindings.dart

```diff
@@ -106,8 +106,8 @@ class _ObjectBoxBindings {
   // query
   obx_query_t obx_query_create;
   obx_query_close_dart_t obx_query_close;
-  obx_query_find_t<Void, int> obx_query_find;
-  obx_query_find_t<Void, int> obx_query_find_ids; // ids = OBX_id_array, else OBX_bytes_array
+  obx_query_find_t<Uint64, int> obx_query_find;
+  obx_query_find_t<Uint64, int> obx_query_find_ids; // ids = OBX_id_array, else OBX_bytes_array
```

Comment:
these types look rather weird - why are they typed the same?

Reply:
Good catch, I had a brainfart there. I thought they were going to be different types, being a `@ffi.Struct` and all that. But my experiment failed.

Reply:
Reverted.